crates/walrus-service/src/test_utils.rs

```diff
@@ -3116,7 +3116,9 @@ pub fn storage_node_config() -> WithTempDir<StorageNodeConfig> {
             admin_socket_path: None,
             node_recovery_config: Default::default(),
             // Uses smaller number of workers in tests to avoid overwhelming the tests.
-            blob_event_processor_config: BlobEventProcessorConfig { num_workers: 3 },
+            blob_event_processor_config: BlobEventProcessorConfig {
+                num_workers: NonZeroUsize::new(3).unwrap(),
```

Comment:
nit: we typically use `expect` and avoid using `unwrap` in non-test code.

Reply:
I thought this was test code. (It's in `test_utils.rs`.)

Reply:
See top of file:

```
#![allow(clippy::cast_possible_truncation, clippy::unwrap_used)]
```